stdlib/json/0/json.rbs

```diff
@@ -560,7 +560,7 @@ class Regexp
   def to_json: (?JSON::State state) -> String
 end
 
-class Set[A]
+class Set[unchecked out A]
```

Comment:
Could you update the type of `json_create?`. It has been broken...

```
def self.json_create: [A] (Hash[String, String | Array[A]] object) -> Set[A]
```

And, the same for other parameterized classes. 🙏

Reply:
done